### DISCUSSION TITLE
我是一只狐狸

### DISCUSSION THREAD

**Discussion by giscus** [darkyzhou/blog-comments#5] [General]
# 我是一只狐狸

在本篇文章中，我将与你一同走过一段自我发现的旅程。我会先从高中时期的挑战讲起，再到大学时期的挫败，之后我会介绍自己如何在反思中发现自己更像是一只狐狸，并以此重塑自信和改变行动模式，在这个被刺猬价值观主导的当代社会中保持自我。希望在读完这篇文章后，你也能发现自己的独特之处，不论是狐狸还是刺猬。

https://darkyzhou.net/articles/i-am-a-fox/

**Comment by wenjia12** [comment]
文笔好好！

**Comment by xianyuYUAN** [comment]
写的好棒

**Comment by atrimydearmome** [comment]
感谢您所作的这篇文章

**Comment by RunningKuma** [comment]
曾一直探寻与自责自己所谓的“做事分心”与“三分钟热度“，感谢您的文章，做一个“狐狸”并不是一个可耻的事情

**Comment by xhj967** [comment]
好有启发呀，我很早就发现，有些人接受新知识快，有些人接受新知识慢，但是却掌握的更牢固

**Comment by lilimul** [comment]
感谢分享.这也体现了泛阅读的好处,如果是自己思考,是比较难逃离这种焦虑的
